### PR TITLE
remove import loadDFG

### PR DESCRIPTION
**Pull request by Affie**

### CODE REVIEW

File: src/FGOSUtils.jl [hunk 1]
```diff
@@ -2,7 +2,7 @@
 #  IIF methods should direclty detect extended types from user import
 # of convert in their namespace
 
-import DistributedFactorGraphs: AbstractPointParametricEst, loadDFG
```
Comment (dehann) [inline src/FGOSUtils.jl:5]: I just added loadDFG yesterday?  What's the issue?

Comment (Affie) [reply src/FGOSUtils.jl:5]: It's not in DFG to import

Comment (dehann) [reply src/FGOSUtils.jl:5]: https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/blob/60ba70fdf026d995e727f72ef021db10835a5c2e/src/FileDFG/services/FileDFG.jl#L171

Comment (dehann) [reply src/FGOSUtils.jl:5]: https://github.com/JuliaRobotics/IncrementalInference.jl/blob/94bc84fd3c22c9d1c8ef09410bd58409c5efc681/src/FGOSUtils.jl#L358

Comment (Affie) [reply src/FGOSUtils.jl:5]: Haha, sneaky. I updated IIF, but DFG was slightly behind.

**Pull request closed without merging**